packages/idyll-components/src/scroller.js

```diff
@@ -3,10 +3,10 @@ const { filterChildren, mapChildren } = require('idyll-component-children');
 import TextContainer from './text-container';
 const d3 = require('d3-selection');
 
-
 const styles = {
   SCROLL_GRAPHIC: {
-    position: 'absolute',
+    position: '-webkit-sticky',
+    position: 'sticky',
```

Comment:
Unfortunately setting multiple styles for a single property doesn't work in React styles like it does in CSS. If you want to set both of these values it has to be done in CSS; although I'm not entirely sure if `-webkit-sticky` is still necessary.

Reply:
Hmmm intriguing -- thanks! Was not aware of that :]

Though I will try to incorporate it still, as according to the [w3 definition of position sticky](https://www.w3schools.com/cssref/pr_class_position.asp), the webkit prefix is required for Safari 6.1 and on.

Reply:
One solution is to toggle a class (e.g. `.idyll-sticky`) in the JavaScript, and then set these styles directly in CSS, this could go in the `idyll-layouts`.

Reply:
Mmk so I tried testing it out in Safari with and without `-webkit-sticky` and Safari's behavior just seems odd, as compared to Chrome. I would describe the behavior as less stable.

But it seems like it's fine without `-webkit-sticky` for now, so I pushed a change to remove it. Though I will look into the toggle class solution you suggested!